<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work</class>
 <widget class="QWidget" name="work">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>ADMUI3Lg</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="suoyan">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="jianshu">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="suoyan0">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>88</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jianshu0">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>88</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="suoyan_text">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jianshu_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="suoyan_num">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jianshu_num">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计件显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>状态显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>模式选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
